worker/worker-lib.ts

```diff
@@ -0,0 +1,59 @@
+import { v4 as uuid } from 'uuid';
+
+// @ts-ignore
```

Comment:
Tried many variations on [worker-loader's advice for using it with TypeScript](https://github.com/webpack-contrib/worker-loader#integrating-with-typescript), but it never bundled correctly (didn't fail out during build, but the bundle would error in my example in the browser). Only ignoring all advice and using the config I've used for other projects with this `@ts-ignore` got it working...

Reply:
There is [this issue](https://github.com/webpack-contrib/worker-loader/issues/221) open on the worker-loader, though that problem isn't exactly what I was seeing.

Reply:
Hmmm.... sounds awful and I'm not at all opposed to just circumventing TS if it winds up causing too much trouble.